mllib/src/main/scala/org/apache/spark/mllib/tree/configuration/BoostingStrategy.scala

```diff
@@ -70,20 +70,31 @@ object BoostingStrategy {
 
   /**
    * Returns default configuration for the boosting algorithm
+   * @param algoStr Learning goal.  Supported:
```

Comment:
This is my bad. In Scala, the parameter name is part of the API. So let's not rename `algo` to `algoStr`.